src/components/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+import arrowmenu from './../../images/arrowmenu.svg';
+import arrowSecond from './../../images/arrow-second.svg';
+
+const Accordion = ({ title, text }) => {
+  const [selected, toggle] = useState(false);
+  return (
+    <div className='accordion'>
+      <div className='title'>
+        <p> {title} </p>
+        <span onClick={() => toggle(!selected)}>
```

Comment:
Please align title and arrows properly 
![Screenshot 2023-01-27 at 16 46 44](https://user-images.githubusercontent.com/85338076/215129926-464da1ef-1984-42d9-8967-a844749a40f1.png)